internal/kafka/go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/twmb/franz-go/pkg/kfake v0.0.0-20250320172111-35ab5e5f5327
 	github.com/xdg-go/scram v1.1.2
 	go.opentelemetry.io/collector/component v1.31.1-0.20250505152726-56c7da210783
+	go.opentelemetry.io/collector/config/configcompression v1.31.0
```

Comment:
Please update the version to v1.31.1-0.20250505152726-56c7da210783 here an in other files